hrms/payroll/doctype/salary_slip/salary_slip.py

```diff
@@ -466,10 +466,11 @@ def get_payment_days(self, joining_date, relieving_date, include_holidays_in_tot
 				return
 
 		end_date = getdate(self.end_date)
+		employee_status = froggy.get_cached_value("Employee", self.employee, "status")
```

Comment:
```suggestion
		employee_status = frappe.get_cached_value("Employee", self.employee, "status")
```

@nomi-gee Salary slips can only be created for employees which don't have the "Left" status. Is this check really necessary?

Reply:
@ruchamahabal then get_employees button should exclude employees to get populated in the payroll with status Left